queries/vala/highlights.scm

```diff
@@ -54,46 +58,56 @@
   "break"
 ] @keyword.return
 
-
 (null) @constant.builtin
 
 [
   "typeof"
   "is"
 ] @keyword.operator
 
+(this) @variable.builtin
+;;; move enum to namespace
+[
+  "enum"
+] @namespace
```

Comment:
I don't think this is `@namespace`

Reply:
> I don't think this is `@namespace`

What is your suggestion, I think it should be another color..or just put it in keyword?

Reply:
> I don't think this is `@namespace`

now it has returned to keyword.